gcore/mdreader/reader_pleiades.cpp

```diff
@@ -422,7 +431,7 @@ char** GDALMDReaderPleiades::LoadRPCXmlFile()
             }
             else {
                  pszValue = CSLFetchNameValue(papszRawRPCList,
-                 CPLSPrintf("ImagetoGround_Values.%s_%d", apszRPCTXT20ValItemsLonLat[i], j));
+                 CPLSPrintf("GroundtoImage_Values.%s_%d", apszRPCTXT20ValItems[i], j));
```

Comment:
Is the change from apszRPCTXT20ValItemsLonLat back to apszRPCTXT20ValItems intended ? If so, the apszRPCTXT20ValItemsLonLat array is now no longer used (compilers complain about that) and should be removed

Reply:
Yes it is intended. Thanks for the explanation. It is my first time with C/C++.. shoud i remove the variable i've added in the gdal_mdreader.h? Because they are not used anymore.. but can be useful in the future

Reply:
I believe it is better to just remove apszRPCTXT20ValItemsLonLat from gcore/mdreader/reader_pleiades.cpp